default/AI/ShipDesignAI.py

```diff
@@ -633,11 +635,11 @@ def evaluate(self):
         # However, we also need to make sure, that the closer we are to requirements,
         # the better our rating is so the optimizing heuristic finds "the right way".
         if self.fuel < self.additional_specifications.minimum_fuel:
-            rating += -1000 * (self.additional_specifications.minimum_fuel - self.fuel)
+            rating += MISSING_REQUIREMENT_MULTIPLIER * (self.additional_specifications.minimum_fuel - self.fuel)
```

Comment:
It probably comes down to readability. Long variable names make it hard to see the structure of a formula and whitespaces make sense in that case in my opinion. Otherwise, I fully agree and will pay more attention to it.